analyzer/pkg/rpm/rpm.go

```diff
@@ -30,6 +30,19 @@ var requiredFiles = []string{
 }
 var errUnexpectedNameFormat = xerrors.New("unexpected name format")
 
+var osVendors = []string{
+	"Amazon Linux",     // Amazon Linux 1
+	"Amazon.com",       // Amazon Linux 2
+	"CentOS",           // CentOS
+	"Fedora Project",   // Fedora
+	"Oracle America",   // Oracle Linux
+	"Red Hat, Inc.",    // Red Hat
+	"AlmaLinux",        // AlmaLinux
+	"CloudLinux",       // AlmaLinux
+	"VMware",           // Photon
+	"openSUSE Project", // OpenSUSE
```

Comment:
I also found `SUSE LLC <https://www.suse.com/>` in `registry.suse.com/suse/sle15:15.3`, but there may be more variety of vendors in SLE. We may want to use `strings.HasPrefix` or `strings.Contains`.

Reply:
Done